### ISSUE TITLE
Example for batch join inconsistent with stream example

### ISSUE THREAD

**Issue by jagare** [influxdata/docs.influxdata.com#1273]
###### URL for relevant page?
https://docs.influxdata.com/kapacitor/v1.3/guides/join_backfill/

###### What products and version are you using?
Kapacitor V1.3

###### Where did you look before opening the issue?
Open issues at https://github.com/influxdata/docs.influxdata.com/issues

Under the heading "Stream method" there is a script "To do the same for the streaming".
However:
It does not calculate 'error_percent' in the same way as for the batch case:` "errors.value" / "views.value"` instead of `"errors.sum" / ("views.sum" + "errors.sum")`.
The resulting field names are not the same.
There is no window defined for the stream script where as in the batch case there is an integration over 1 minute.

In summary: the streaming script doesn't do the same thing at all.

Then the name percentage is given to something that is actually a ratio. Multiply by 100 or call it ratio.
